action.yml

```diff
@@ -21,5 +21,5 @@ branding:
   icon: activity
 
 runs:
-  using: node12
+  using: node18
```

Comment:
node18 is not supported for github actions. latest version for github actions is node16

Reply:
ok @tuunit